packages/i18n/src/load-i18n.ts

```diff
@@ -110,6 +107,56 @@ const getAppKitChunkImport = (locale: string): Promise<AppKitImportData> => {
   }
 };
 
+const getCommunityKitChunkImport = async (
```

Comment:
Just thinking out loud: in the future each package has it's own loadTranslationsChunkByLocale. While then AppKit offers a registerTranslationsChunk which is then called in each application for each loader? Breaking change, more on consumers but decouples things. Or did you already think of something else?

Reply:
Maybe something like that yes.

I don't have any specific idea at the moment. We need to come up with a scalable and easy-to-use approach I would say.